retworkx/__init__.py

```diff
@@ -750,6 +755,9 @@ def is_isomorphic(
     :param bool id_order: If set to ``False`` this function will use a
         heuristic matching order based on [VF2]_ paper. Otherwise it will
         default to matching the nodes in order specified by their ids.
+    :param int call_limit: An optional bound on the number of states that VF2
+        algorithm visits while searching for a solution. If it exceeds this limit,
+        the algorithm will stop and return ``False``. Default: ``None``.
```

Comment:
Default `None` here means there is no limit right? It's just a bit ambiguous, I'd probably just drop it since it's covered by the optional part:

```suggestion
        the algorithm will stop and return ``False``.
```

Reply:
Yes, `None` means no limit.